packages/lodestar-types/src/ssz/generators/api.ts

```diff
@@ -15,4 +15,29 @@ export const ForkResponse = (ssz: IBeaconSSZTypes): ContainerType => new Contain
     fork: ssz.Fork,
     chainId: ssz.Uint64
   }
-});
\ No newline at end of file
+});
+
+export const AttesterDuty = (ssz: IBeaconSSZTypes): ContainerType => new ContainerType({
+  fields: {
+    validatorPubkey: ssz.BLSPubkey,
+    aggregatorModulo: ssz.Number64,
+    committeeIndex: ssz.CommitteeIndex,
+    attestationSlot: ssz.Slot,
+  },
+});
+
+export const SyncingStatus = (ssz: IBeaconSSZTypes): ContainerType => new ContainerType({
+  fields: {
+    startingBlock: ssz.Uint64,
+    currentBlock: ssz.Uint64,
+    highestBlock: ssz.Uint64,
+  },
+});
+
+export const ValidatorResponse = (ssz: IBeaconSSZTypes): ContainerType => new ContainerType({
+  fields: {
+    ...ssz.Validator.fields,
+    index: ssz.ValidatorIndex
```

Comment:
`index` should be declared before `...Validator.fields`

Reply:
any particular reason?

Reply:
The order is important for ssz serialization